src/main/java/io/teiler/server/Tylr.java

```diff
@@ -15,6 +15,11 @@
 
     public static void main(String[] args) {
         SpringApplication.run(Tylr.class, args);
+    
+        String test = null;
+        if (test == null) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2589)